.github/workflows/qunit_tests-additional-renovation.yml

```
@@ -0,0 +1,186 @@
+name: Additional QUnit
+
+concurrency:
+  group: wf-${{github.event.pull_request.number}}-${{github.workflow}}
+  cancel-in-progress: true
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    timeout-minutes: 60
+
+    steps:
+    - name: Get sources
+      uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '15'
+
+    - name: Restore npm cache
+      uses: actions/cache@v2
+      with:
+        path: ./node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: ${{ runner.os }}-node-modules
+
+    - name: Run npm install
+      run: npm install --no-audit --no-fund
+
+    - name: Build
+      env:
+        DEVEXTREME_TEST_CI: "true"
+        DOTNET_CLI_TELEMETRY_OPTOUT: "true"
+        DOTNET_SKIP_FIRST_TIME_EXPERIENCE: "true"
+      run: npm run build
+      
+    - name: Zip artifacts 
+      run: |
+        7z a -tzip -mx3 -mmt2 artifacts.zip artifacts scss/bundles testing/tests/Renovation/widgets.json
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: devextreme-artifacts
+        path: artifacts.zip
+        retention-days: 1
+
+  qunit-tests:
+    needs: build
+    runs-on: ubuntu-latest
+    name: Constel ${{ matrix.CONSTEL }}
```

Several variants here:
First will produce the dash-separated string made of object fields like
`misc-PST8PDT`, `ios9-ui.editors-true`, `android6-true-ui.editors-true`. Not very informative, but compact.
```suggestion
    name: ${{join(matrix.envs.*,'-')}}
```
---

The second will stringify the JSON (it will actually print the param as-is):
` { CONSTEL: "misc", TZ: 'PST8PDT' }`, `{ MOBILE_UA: "ios9", CONSTEL: "ui.editors", NO_HEADLESS: true }`. Takes a lot of space, but informative
```suggestion
    name: ${{toJSON(matrix.envs)}}
```

---
You can also add the `name` field to a matrix argument, and print it:
```suggestion
    name: ${{matrix.envs.name}}
```